napari/layers/intensity_mixin.py

```diff
@@ -70,36 +77,52 @@ def contrast_limits(self):
 
     @contrast_limits.setter
     def contrast_limits(self, contrast_limits):
-        validate_2_tuple(contrast_limits)
-        self._contrast_limits_msg = (
-            format_float(contrast_limits[0])
-            + ', '
-            + format_float(contrast_limits[1])
-        )
-        self.status = self._contrast_limits_msg
-        self._contrast_limits = contrast_limits
+        self.events.contrast_limits(contrast_limits)
+
+    def _on_contrast_limits_change(self, value):
+        """Set the contrast limits.
+
+        Parameters
+        ----------
+        value : tuple
+            Contrast limits, (min, max).
```

Comment:
Yeah, the more I read of this PR, the more I'm disliking "unifying on 'value'", which is sounding more and more like "unifying on 'x'". I don't necessarily want to hold it up, but I want us to think about reverting and calling the event values by what they actually hold for each event.

Reply:
I'm happy to revert. I agree for long functions it's confusing, particularly now when we don't really use `value`. For short ones I think it's ok, but I'd change them too for consistency. Thoughts @tlambert03? If you have no objections I'll change them all now